ggshield/cmd/iac/scan/diff.py

```diff
@@ -111,9 +112,21 @@ def iac_scan_diff(
 ) -> Optional[IaCDiffScanResult]:
     config = ctx.obj["config"]
     client = ctx.obj["client"]
-
+    verbose = config.user_config.verbose if config and config.user_config else False
+    if verbose:
+        click.echo(f"> Scanned files in reference {ref}")
+        for filepath in get_iac_filepaths(directory, ref):
+            click.echo(f"- {click.format_filename(filepath)}", err=True)
+        click.echo("")
     reference_tar = get_iac_tar(directory, ref)
     current_ref = INDEX_REF if include_staged else "HEAD"
+    if verbose:
+        if include_staged:
+            click.echo(f"> Scanned files in current state (staged)")
+        else:
+            click.echo(f"> Scanned files in current state")
+        for filepath in get_iac_filepaths(directory, current_ref):
+            click.echo(f"- {click.format_filename(filepath)}", err=True)
```

Comment:
Why is it sometimes in `err` and sometimes not ?

Reply:
I think log messages should be displayed on stderr. I have set `err=True` on all of those messages.